debian/kubelet/kubelet-wrapper

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source /etc/default/kubelet
```

Comment:
I'd expect this to be `EnvironmentFile=/etc/default/kubelet` in `kubelet.service`, unless we think that users should be really allowed to do anything bash permits?